regparser/web/jobs/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+
+class ParsingJob(models.Model):
+
```

Comment:
Something to consider here -- you might not want to track all these fields in a separate model. It limits reusability; for example, we can't reuse this model to represent a `proposal_pipeline` job or a simpler `write_to` job. You can also pull this information from the `job` in Redis (until that job's deleted).